requirements.txt

```diff
@@ -1,18 +1,18 @@
-dimod==0.12.20
-dwave-optimization==0.4.0
-dwave-preprocessing==0.6.7
-dwave-cloud-client==0.12.0
+dimod==0.12.21
+dwave-optimization==0.6.4
```

Comment:
Why not the latest 0.6.7?

Reply:
We usually pin to oldest supported versions in requirements.txt (for testing).